test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/PropagationTestHelpers.cs

```diff
@@ -0,0 +1,53 @@
+using System.Globalization;
+using Datadog.Trace.Propagation;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.Helpers
+{
+    public static class PropagationTestHelpers
+    {
+        public static void AssertPropagationEnabled(MockTracerAgent.Span expectedSpan, ProcessResult processResult)
+        {
+            // Verify DD headers
+            var ddTraceId = StringUtil.GetHeader(processResult.StandardOutput, DDHttpHeaderNames.TraceId);
+            var ddParentSpanId = StringUtil.GetHeader(processResult.StandardOutput, DDHttpHeaderNames.ParentId);
+
+            Assert.Equal(expectedSpan.TraceId.ToString(), ddTraceId);
+            Assert.Equal(expectedSpan.SpanId.ToString(CultureInfo.InvariantCulture), ddParentSpanId);
+
+            // Verify B3 headers
+            var b3TraceId = StringUtil.GetHeader(processResult.StandardOutput, B3HttpHeaderNames.B3TraceId);
+            var b3SpanId = StringUtil.GetHeader(processResult.StandardOutput, B3HttpHeaderNames.B3SpanId);
+            var b3ParentSpanId = StringUtil.GetHeader(processResult.StandardOutput, B3HttpHeaderNames.B3ParentId);
+
+            Assert.Equal(expectedSpan.TraceId.ToString(), b3TraceId); // TODO: With DD Trace ID Convention, it will be in DD Trace ID format
```

Comment:
Constructing TraceId in constructor causes problems here. 64bit key will be always in line with DD convention.